scripts/submit-analytics.js

```diff
@@ -122,6 +124,32 @@ function getRealmVersion() {
   return packageJson["version"];
 }
 
+/**
+ * Reads and parses `dependencies.list`.
+ * Each line has be form "KEY=VALUE", and we to find "REALM_CORE_VERSION"
+ *
+ * @returns the Realm Core version as a string
+ */
+function getRealmCoreVersion() {
+  const dependenciesListPath = path.resolve(__dirname, "../dependencies.list");
+  const dependenciesList = fse
+    .readFileSync(dependenciesListPath)
+    .toString()
+    .split("\n")
+    .map((s) => s.split("="));
+  return dependenciesList.find((e) => e[0] === "REALM_CORE_VERSION")[1];
+}
+
+/**
+ * Determines if `npm` or `yarn` is used.
+ *
+ * @returns An array with two elements: method and version
```

Comment:
Not sure if it's necessary to mention what I added below. (Saw it from [this npm package](https://www.npmjs.com/package/npm-config-user-agent-parser?activeTab=readme).)
```suggestion
 * @returns An array with two elements: method and version. (The version for npm will be "?" if npm is not used).
```

Reply:
I believe it will also return version if you are using yarn

Reply:
Good point!